tests/test_sonoff_basiczbr3.py

```diff
@@ -0,0 +1,72 @@
+"""Tests for Sonoff BASICZBR3 quirk."""
+
+import zhaquirks
+from zhaquirks.const import (
+    CLUSTER_ID,
+    COMMAND,
+    ENDPOINT_ID,
+    LONG_PRESS,
+    PARAMS,
+    SHORT_PRESS,
+)
+from zhaquirks.sonoff.basiczbr3 import FLASH, POWER_ON
+
+zhaquirks.setup()
+
+
+def test_basiczbr3_quirk_registered():
+    """Test that the quirk is properly registered."""
+
+    from zigpy.quirks import DEVICE_REGISTRY
+
+    import zhaquirks.sonoff.basiczbr3  # noqa: F401
+
+    quirks = DEVICE_REGISTRY.registry_v2.get(("SONOFF", "BASICZBR3"))
+    assert quirks is not None, "BASICZBR3 quirk not found in registry"
+    assert len(quirks) == 1, "Expected exactly one quirk for BASICZBR3"
+
+
+def test_basiczbr3_device_automation_triggers():
+    """Test that device automation triggers are correctly defined."""
+
+    from zigpy.quirks import DEVICE_REGISTRY
+
+    import zhaquirks.sonoff.basiczbr3  # noqa: F401
+
+    quirks = DEVICE_REGISTRY.registry_v2.get(("SONOFF", "BASICZBR3"))
+    assert quirks is not None, "BASICZBR3 quirk not found in registry"
+
+    quirk_metadata = quirks[0]
+    triggers = quirk_metadata.device_automation_triggers_metadata
+
+    # Expected triggers
+    expected_triggers = {
```

Comment:
As a unit test, this really doesn't do much: the triggers are defined in the quirk and this is sort of a copy/paste.